harvard-library/librarycloud_collections
# Ltsmaint 504

**Patching critical, high vulnerabilities**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/LTSMAINT-504)

# What does this Pull Request do?
Upgrades pom.xml with patched versions of software with critical and high vulnerabilities; also able to patch all moderate vulnerabilities as well. Small code changes in a couple classes to upgrade from java lang to commons lang3. Upgrades readme and props example files

# How should this be tested?

Build and test locally (This branch builds locally and runs in local tomcat, tested by Michael)
- Clone this branch locally
- See `Testing app locally` in README.md
Note, Tests in CollectionsDAO set to @ignore for now, a legacy issue and not part of security upgrade

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N/A, version upgrades only, no functional changes
- integration tests? No

# Interested parties
Tag (@ mention) interested parties
